.github/ISSUE_TEMPLATE/giskard_bug_template.yaml

```diff
@@ -0,0 +1,136 @@
+
+  # - type: markdown
+  #   attributes:
+      # value: | 
+      #   You can collect some of this information using our environment capture [script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)  You can also obtain the TensorFlow version with: <br> 1. TF 1.0: `python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"` <br>2. TF 2.0: `python -c "import giskard ; print(tf.version.GIT_VERSION, tf.version.VERSION)"`
+
+  - type: dropdown
+    id: source
+    attributes:
+      label: Source
+      description: Giskard installed from
+      options:
+        - source
+        - binary
+    validations:
+      required: true
+  - type: input
+    id: giskardversion
+    attributes:
+      label: Giskard Version
+      description: 
+      placeholder: ex,. giskard 1.3
+    validations:
+      required: true
+  - type: dropdown
+    id: Code
+    attributes:
+      label: Custom Code
+      description: 
+      options:
+        - "Yes"
+        - "No"
+    validations:
+      required: true
+  - type: input
+    id: OS
+    attributes:
+      label: OS Platform and Distribution
+      description: 
+      placeholder: e.g., Linux Ubuntu 16.04
+    validations:
+      required: false
+  # - type: input
+  #   id: Mobile
+  #   attributes:
+  #     label: Mobile device
+  #     description: 
+  #     placeholder: e.g., Linux Ubuntu 16.04
+  #   validations:
+  #     required: false
+  - type: input
+    id: Python
+    attributes:
+      label: Python version
+      description: 
+      placeholder: e.g., 3.9
+    validations:
+      required: false
+  # - type: input
+  #   id: Bazel
+  #   attributes:
+  #     label: Bazel version
+  #     description: if compiling from source
+  #     placeholder: 
+  #   validations:
+  #     required: false
+  # - type: input
+  #   id: Compiler
+  #   attributes:
+  #     label: GCC/Compiler version
+  #     description: if compiling from source
+  #     placeholder: 
+  #   validations:
+  #     required: false
+  # - type: input
+  #   id: Cuda
+  #   attributes:
+  #     label: CUDA/cuDNN version
+  #     description:
+  #     placeholder: 
+  #   validations:
+  #     required: false
+  # - type: input
+  #   id: Gpu
+  #   attributes:
+  #     label: GPU model and memory
+  #     description: if compiling from source
+  #     placeholder: 
+  #   validations:
+  #     required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Current Behaviour?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: code-to-reproduce
+    attributes:
+      label: Standalone code to reproduce the issue
```

Comment:
Standalone code or list of steps ? As we aren't just a library but also have a user interface, some bugs may not require any code to happen